internal/runners/clean/run_win.go

```diff
@@ -27,7 +27,7 @@ func (u *Uninstall) runUninstall() error {
 		return err
 	}
 
-	u.out.Print(locale.T("clean_success_message"))
+	u.out.Print(locale.Tl("clean_message", "Attempting to remove State Tool and related files"))
```

Comment:
This is still lacking UX. If we have to defer the deletion then we should at least tell the user how to find out if and why it failed.

Reply:
I don't think we're able to tell them exactly why it failed as this script will execute after the state tool process has completed.  I've tried to pass off `stdout` to the script and release the process but it appears that that will keep that `state.exe` process alive. Added some more information to the uninstall messaging for likely reasons why uninstall may fail and also how to verify that it actually removed the install.

Reply:
Can we have it write to a log file, and tell the user where it's located?

Reply:
💡